src/less/switch/base/switch.less

```diff
@@ -42,7 +42,8 @@ span.switch__button {
     }
 }
 
-input.switch__control {
+input.switch__control,
+span.switch__control {
```

Comment:
Can we simplify this to 1 selector by removing input and span? Definitely not required, but I'm curious if we do it this way for a reason.

Reply:
We could, but we are trying to limit which tags can be used for a11y reasons. Believe it or not, but I've seen some devs here use `<i>` for everything!!!

In an ideal world we would probably enforce this through process and tooling, without impacting our CSS selectors, but we are not quite there yet. This for now is the quickest way of reducing our a11y bugs on the site.